src/Config/Config.py

```diff
@@ -88,3 +89,28 @@ def remove_proxy(self):
             config_file.seek(0)
             config_file.truncate()
             config_file.write(json.dumps(config))
+
+    def set_user(self, user):
+        address='http://codeforces.com/api/user.info?handles='+user
+        response = requests.get(url=address,proxies=self.get_proxy())
+        if(response.status_code!=200):
+            raise UsernameError('User not Found')
```

Comment:
No implementation of UsernameError class. Also I think "UsernameNotFound" would be a better name